src/plugins/intel_cpu/src/nodes/qkv_proj.cpp

```diff
@@ -443,6 +443,11 @@ bool QKVProjection::isSupportedOperation(const std::shared_ptr<const ov::Node>&
                 return false;
             }
 
+            if (config.hidden_size < CACHE_BLK_K_SIZE * 8) {
```

Comment:
The threashold equals 2048. Isn't it too agressive? Won't we have regressed models, e.g. SLMs?

Reply:
I tested this PR, using some SLM on my local SPR, with prompt-length 9 and batch-size 512, we can see 2048 is roughly the proper threshold

-  qwen2-1.5b with hidden_size==1536, 437.8 ms +   86.6 ms ==(after-fused)==> 460.8 ms +   87.6 ms 
-  llama-3.2-1b-instruct with hidden_size==2048,    379.2 ms +   67.9 ==(after-fused)==> 386.2 ms +   66.3 ms
-  llama-2-7b with hidden_size==4096, 2227.6 ms +  276.1 ms ==(after-fused)==> 2218.7 ms +  265.7 ms